Travis CI build failing on node.js v0.8 with npm failure

The Travis CI build is failing for node.js v0.8.  Here is the error output: 

```
npm ERR! peerinvalid The package npmconf does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer npm-registry-client@3.2.2 wants npmconf@^2.1.0

npm ERR! System Linux 2.6.32-042stab090.5
npm ERR! command "/home/travis/.nvm/v0.8.28/bin/node" "/home/travis/.nvm/v0.8.28/bin/npm" "install" "-g" "npm"
npm ERR! cwd /home/travis/build/simondean/cucumber-js
npm ERR! node -v v0.8.28
npm ERR! npm -v 1.2.30
npm ERR! code EPEERINVALID
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR! /home/travis/build/simondean/cucumber-js/npm-debug.log
npm ERR! not ok code 0

The command "npm install -g npm" failed and exited with 1 during .

Your build has been stopped.
```

The issue seems to be related to commit e71f549eff565ada14dd8bc4c3898aac4e6e3672 and also this npm issue https://github.com/npm/npm/issues/6246

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,7 @@
 rvm: "1.9.2"
 before_install:
-  - npm install -g npm
+  - '[ "${TRAVIS_NODE_VERSION}" != "0.8" ] || npm install -g npm@1.4.28'
+  - npm install -g npm@latest
   - ruby --version
   - gem --version
   - rvm --version

